pkg/controller/performanceprofile/performanceprofile_controller.go

```diff
@@ -275,11 +276,19 @@ func (r *ReconcilePerformanceProfile) Reconcile(request reconcile.Request) (reco
 	// apply components
 	result, err := r.applyComponents(instance)
 	if err != nil {
-		klog.Errorf("failed to deploy components: %v", err)
+		klog.Errorf("failed to deploy performance profile %q components: %v", instance.Name, err)
 		return reconcile.Result{}, err
```

Comment:
we shouldn't be breaking out early here. If an error occurs during ```applyComponents``` we would treat that as a degraded condition and want to report the error msg in the condition.

Reply:
done